<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptoOperationsConfigWidget</class>
 <widget class="QWidget" name="CryptoOperationsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>398</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="emailOperationsTab">
      <attribute name="title">
       <string>EMail Operations</string>
      </attribute>
      <layout class="QVBoxLayout" name="subverticalLayout">
       <item>
        <widget class="QCheckBox" name="quickSignCB">
         <property name="whatsThis">
          <string comment="@info:whatsthis">When 'Quick Mode' is enabled, no dialog is shown when signing (encrypting) emails, respectively, unless there is a conflict that needs manual resolution.</string>
         </property>
         <property name="text">
          <string>Use 'Quick Mode' when signing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="quickEncryptCB">
         <property name="whatsThis">
          <string comment="@info:whatsthis">When 'Quick Mode' is enabled, no dialog is shown when signing (encrypting) emails, respectively, unless there is a conflict that needs manual resolution.</string>
         </property>
         <property name="text">
          <string>Use 'Quick Mode' when encrypting</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fileOperationsTab">
      <attribute name="title">
       <string>File Operations</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="whatsThis">
          <string comment="@info:whatsthis">&lt;p&gt;Choose here which of the configured checksum programs should be used when creating checksum files.&lt;/p&gt;
&lt;p&gt;When verifying checksums, the program to use is automatically found, based on the names of the checksum files found.&lt;/p&gt;
</string>
         </property>
         <property name="text">
          <string>Checksum program to use:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="checksumDefinitionCB">
         <property name="whatsThis">
          <string comment="@info:whatsthis">&lt;p&gt;Choose here which of the configured checksum programs should be used when creating checksum files.&lt;/p&gt;
&lt;p&gt;When verifying checksums, the program to use is automatically found, based on the names of the checksum files found.&lt;/p&gt;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
